canonical-web-and-design/ubuntu.com
# Replace FBCLID__c to Facebook_Click_ID__c on forms

## Done

- Due to a misnaming in SFDC, we need to update the name of the facebook click id field
- Replace FBCLID__c to Facebook_Click_ID__c on forms

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and test a few forms
- Make sure that the forms all have the field `FBCLID__c` replaced with `Facebook_Click_ID__c`
